examples/gno.land/r/nt/boards2/v1/freeze.gno

```diff
@@ -0,0 +1,76 @@
+package boards2
+
+import "std"
+
+type freezeArgs struct {
+	boardID  BoardID
+	threadID PostID
+	replyID  PostID
+}
+
+func (args freezeArgs) getThread() (*Post, *Board) {
+	board := mustGetBoard(args.boardID)
+	assertBoardIsNotFrozen(board)
+
+	thread := mustGetThread(board, args.threadID)
+	return thread, board
+}
+
+func (args freezeArgs) getReply() (*Post, *Board) {
+	board := mustGetBoard(args.boardID)
+	assertBoardIsNotFrozen(board)
+
+	thread := mustGetThread(board, args.threadID)
+	assertThreadVisible(thread)
+	assertThreadIsNotFrozen(thread)
+
+	reply := mustGetReply(thread, args.replyID)
+	assertReplyVisible(reply)
+	return reply, board
+}
```

Comment:
I'm wondering why not just keep using `mustGetBoard`, `mustGetThread` and `mustGetReply` instead of using a type that wraps them 🤔. Wrapping might make auditing the realm code harder by making the code less explicit. WDYT @x1unix?

Otherwise it would make sense to just return one type per function:
- freezeArgs.getThread() -> *Post
- freezeArgs.getReply() -> *Post
- freezeArgs.getBoard() -> *Board